runpod.sh

```
@@ -0,0 +1,37 @@
+# Based on https://github.com/abideenml/AutoOPRO/tree/master
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)